src/main/java/core/basesyntax/Model/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.Model;
+
+import core.basesyntax.Base.Machine;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");
```

Comment:
Similarly to the `doWork` method, the message 'Truck stopped working' should be more informative. Indicate what type of Machine has stopped working by including model or unique characteristics of the Truck.